schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+ CREATE TABLE animals (
+ ID                              INT               PRIMARY KEY       NOT NULL,
+ NAME                       CHAR(50)                                   NOT NULL,
+ DATE_OF_BIRTH       DATE                                           NOT NULL,
+ ESCAPE_ATTEMPTS   INT                                             NOT NULL,
+ NEUTERED                BIT                                              NOT NULL,
+ WEIGHT_KG              DECIMAL
+ );
```

Comment:
- [ ] For the `neutered` variable Please use a data type that indicates a true or false value like `boolean` instead of `bit` because in PostgreSQL `bit` data type is used for fixed-length bit string.
- [ ] [Optional]Kindly delete all white spaces to make your code more readable.